.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,35 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the Tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone the expirationd module
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/expirationd
+
+      - name: Download the Tarantool build artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+
+      - name: Install Tarantool
+        # Now we're lucky: all dependencies are already installed. Check package
+        # dependencies when migrating to other OS version.
+        run: >
+          sudo dpkg -i tarantool_*.deb tarantool-common_*.deb
+          tarantool-dev_*.deb
```

Comment:
Why the list is split for two lines?

Reply:
For more readability. I try to keep the length of the line equal to 80 characters.

Reply:
If so, then we need to fix the indentation for better readability :)

```
          sudo dpkg -i tarantool_*.deb tarantool-common_*.deb
                       tarantool-dev_*.deb
```

Reply:
May be
```
sudo dpkg -i tarantool_*.deb tarantool-common_*.deb 
    tarantool-dev_*.deb
```
?

Reply:
Package names should start at the same position on both lines.

Reply:
Fixed.